virtcontainers/network.go

```diff
@@ -226,7 +225,7 @@ func networkLogger() *logrus.Entry {
 func (endpoint *VirtualEndpoint) Attach(h hypervisor) error {
 	networkLogger().WithField("endpoint-type", "virtual").Info("Attaching endpoint")
 
-	if err := xconnectVMNetwork(&(endpoint.NetPair), true); err != nil {
+	if err := xconnectVMNetwork(&(endpoint.NetPair), true, h.hypervisorConfig().NumVCPUs); err != nil {
```

Comment:
Do we care about hotplugged CPUs? Properly not since they might be unplugged later. Just want to know your opinions about it.

Reply:
Hey @bergwolf  - under what use-cases would we be actively hotplugging/unplugging?

Reply:
@egernst Currently no. I don't know any use case that would require removing a container while having other containers still running in the same sandbox. It is just that our APIs and CLI interfaces allow such behavior. nvm. It's fine with me to just use the default CPU numbers for the queue size.

Reply:
Well, I think we *do* care about the resulting number of vCPUS.  This shouldn't default to the default CPUs, but to what we request.

@amshinde @sboeuf